pkg/image/qemu.go

```diff
@@ -120,3 +142,14 @@ func ConvertQcow2ToRawStream(url *url.URL, dest string) error {
 func Validate(image, format string) error {
 	return qemuIterface.Validate(image, format)
 }
+
+func reportProgress(line string) {
+	// (45.34/100%)
+	matches := re.FindStringSubmatch(line)
+	if len(matches) == 2 {
```

Comment:
The owner uid (I should name it that) is either the PVC UID or DV UID that owns the PVC. There won't be a possibility of multiples.

Reply:
The UID string is turned into a label for use by the UI to correlate the progress with a particular entity, if its empty, its empty and we can't correlate, but it won't break anything.

Reply:
Maybe it won't break anything, but I'm thinking why do anything prometheus related if the string is empty?  Which would be the case with upload.

Reply:
Well maybe we should do something prometheus related for upload, if we have the content length of course.

Reply:
To get back to the question at hand, if we don't have the UID, which is possible, we still report the progress, it is just harder for whomever is looking at it to correlate the progress to a particular entity. That is the main reason we are putting the UID in a label in the first place, so for instance the UI can take the UID, and look up the entity and display a progress bar for it or something.

Reply:
I disagree, the UID is just a hint for the UI so it can correlate if it wants, but having the progress available without the UID is still useful information to have.